src/ast.mli

```diff
@@ -1,4 +1,4 @@
-type program = None
+type program = message list * actor list * func list
```

Comment:
this seems right, but why don't we have a `string list` tuple in this definition?